idea/meta-storage-2/src/server.ts

```diff
@@ -0,0 +1,25 @@
+import { logger } from '@gear-js/common';
+import { AppDataSource } from './database';
+import { main } from './main';
+import { MetaService } from './service';
+import config from './config';
+
+const port = config.server.port;
+
+const initMetaService = () => {
+  logger.info('Connected to the database');
+  return new MetaService();
+};
+
+AppDataSource.initialize()
+  .then(initMetaService)
+  .then(main)
+  .then((app) => {
+    app.listen(port, () => {
+      logger.info(`Server is running on port ${port}`);
+    });
+  })
+  .catch((error) => {
+    logger.error("Can't start the indexer", { error, stack: error.stack });
+    process.exit(1);
+  });
```

Comment:
I'd suggest to move all this stuff to `main.ts` and initialise all services in `main` function

Reply:
In this case, I won't be able to properly friend the services due to the lack of dependency injection capability. Alternatively, I could rename the main file to avoid confusion

Reply:
Yes let's rename then